docs/source/references/functional.rst

```diff
@@ -382,3 +382,8 @@ wer [func]
 
 .. autofunction:: torchmetrics.functional.wer
     :noindex:
+
+bert_score [func]
```

Comment:
could you move this to be the first metric in the text section (alphabetical order)?